tests/test_tui_api.py

```diff
@@ -5,12 +5,9 @@
 from ansys.fluent.core.services.datamodel_tui import TUIMenu
 
 
-@pytest.mark.skip("randomly failing due to missing transcript capture")
+@pytest.mark.skip("Failing in github")
```

Comment:
This one is passing locally in both standalone and container mode, not sure why it is failing only on github

Reply:
Also works fine here in Windows and Linux inside/outside containers, very odd

Few ideas below, don't know if you've already tried these or not (we can also try to fix this later in a separate PR)

Reply:
I have tried the delay in the past, but it didn't help. The transcript output should be available in stdout. I think there is some issue in the streaming service which is manifested only in a github runner (in an environment with limited resource perhaps). We'll try to investigate it separately.